resolwe_bio/processes/chip_seq/macs14.yml

```diff
@@ -38,14 +38,34 @@
     - name: peaks_bed
       label: Peaks (BED)
       type: basic:file
+    - name: summits_bed
+      label: Summits (BED)
+      type: basic:file
     - name: peaks_xls
       label: Peaks (XLS)
       type: basic:file
     - name: wiggle
       label: Wiggle
       type: basic:file
-    - name: summits_bed
-      label: Summits (BED)
+    - name: control_bigwig_all_gbrowsers
```

Comment:
This should be `control_bigwig` ( and `treat_bigwig` bellow) to keep the name more consise